DigitalLearningSolutions.Data/DataServices/RegistrationDataService.cs

```diff
@@ -21,18 +21,21 @@ public class RegistrationDataService : IRegistrationDataService
         private readonly IDbConnection connection;
         private readonly IUserDataService userDataService;
 
-        public RegistrationDataService(IDbConnection connection)
+        public RegistrationDataService(IDbConnection connection, IUserDataService userDataService)
         {
             this.connection = connection;
+            this.userDataService = userDataService;
         }
 
         public string RegisterNewUserAndDelegateAccount(DelegateRegistrationModel delegateRegistrationModel)
         {
-            // TODO HEEDLS-886: this method previously returned error codes as well as candidate numbers.
+            // TODO HEEDLS-900: this method previously returned error codes as well as candidate numbers.
             // any code that calls it and handled those errors on the basis of the codes needs to be updated
             connection.EnsureOpen();
             using var transaction = connection.BeginTransaction();
 
+            var currentTime = DateTime.UtcNow;

```

Comment:
We should use ClockService.UtcNow rather than DateTime.UtcNow.

Reply:
https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/20067450883/Use+ClockService+instead+of+DateTime.UtcNow